hail/src/main/scala/is/hail/cxx/Compile.scala

```diff
@@ -24,14 +26,14 @@ object Compile {
     tub.include("hail/Utils.h")
     tub.include("hail/Region.h")
     tub.include("hail/Upcalls.h")
+    tub.include("hail/Hadoop.h")
     tub.include("hail/SparkUtils.h")
     tub.include("hail/ObjectArray.h")
 
     tub.include("<cstring>")
 
     val fb = tub.buildFunction(tub.genSym("f"),
-      (("SparkFunctionContext", "ctx") +: args.map { case (_, typ) =>
-        typeToCXXType(typ) -> "v" }).toArray,
+      (("SparkFunctionContext", "ctx") +: ("HadoopConfig", "hadoop_config") +: args.map { case (_, typ) => typeToCXXType(typ) -> "v" }).toArray,
```

Comment:
hmm... I had an idea that we would just put the hadoop config to the SparkFunctionContext struct to avoid having to pass more arguments for the Emit function and everything was in one place, but I guess I'm also not super attached to this.